### ISSUE TITLE
Demonstration issues

### ISSUE THREAD

**Issue by nielsdrost** [c3s-magic/c3s-magic-frontend#74]
Must be completed before the demo
- [x] Complete 90% of the set of diagnostic pages
- [x] Add Home Page with some text and the interactive map from the anomaly agreement diagnostic
- [x] Add the /#/adagucviewer page to the menu
- [x] Add a link to the :8080/.... advanced explorer to the adagucviewer page

Stretch goals
- [x] Add new dataset by Nina to the advanced explorer
- [x] Make the interactive maps in the diagnostics bigger
- [x] Add a download button for data and map of the current view to the diagnostic pages

**Comment by maartenplieger** [comment]
Working on home page atm

**Comment by maartenplieger** [comment]
Added adaguc viewer to menu:
![c3s-magic-adaguc-viewer](https://user-images.githubusercontent.com/8201298/40907326-c4a59d5e-67e3-11e8-9044-aa4c5e40ade5.png)

**Comment by maartenplieger** [comment]
Added link to advanced viewer.
![link_to_adaguc-viewer](https://user-images.githubusercontent.com/8201298/40909665-6701644c-67ea-11e8-92fc-ef22bb87c42d.png)

Could add opening of current dataset automatically, but that is not done yet.

**Comment by fdiblen** [comment]
Backend is almost ready. Closing the issue.
